packages/manager/modules/pci/src/components/project/empty/empty.html

```diff
@@ -0,0 +1,43 @@
+<div class="container-fluid px-0 pci-project-empty">
+    <div class="row">
+        <div class="col-md-3">
+            <div class="p-5 placeholder">
+                <img data-ng-if="$ctrl.imageSource"
+                    data-ng-src="{{:: $ctrl.imageSource }}"
+                    data-ng-attr-alt="{{:: $ctrl.imageAlt }}"
+                    alt=""
+                    class="w-100">
+               <img data-ng-if="!$ctrl.imageSource"
+                    data-ng-src="{{:: $ctrl.placeholder }}"
+                    class="w-100"
+                    alt="">
+            </div>
+        </div>
+        <div class="col-md-9" data-ng-transclude></div>
+    </div>
+
+    <div class="row pt-5">
+        <div class="col-sm-6 col-md-4 mb-4" data-ng-repeat="guide in $ctrl.guides track by $index">
```

Comment:
```suggestion
        <div class="col-sm-6 col-md-4 mb-4" data-ng-repeat="guide in ::$ctrl.guides track by $index">
```

Reply:
I don't know if it makes sense here. Let's suppose the `guides` binding changes, those changes won't be reflected here

Reply:
Is it the case right now ? Because if it's not, it would mean we should never use `::` or I don't understand your point :)